src/lib/downsampleMp3.js

```diff
@@ -20,7 +25,13 @@ export default (inputFile) => {
         compressionQuality: 5 - 96 // see `man soxformat` search for '-C' for more info
       })
 
-    job.on('error', reject)
+    job.on('error', () => {
+      reject(new Error(clk.red(`Failed to convert file: ${inputFile} \n retrying...`)))
```

Comment:
here you're rejecting the promise thus fulfilling it (a promise can only resolve or reject once), I don't think this retry would work with the rest of the code